app/views/provider_mailer/_unsubscribe.text.erb

```diff
@@ -0,0 +1,5 @@
+# Change your email notification settings
+
+You can turn off email notifications if you do not want to be told when application statuses change:
+
+<%= HostingEnvironment.application_url %>/provider/account/notification-settings
```

Comment:
Any reason not to use the `_url` helper here?

Reply:
Good point will update :+1:

Reply:
@duncanjbrown https://github.com/DFE-Digital/apply-for-teacher-training/pull/3727/commits/9bd4d5c1aa2b60e9fb8d40f91fe1b95b3c5d9fd0#diff-c01bc21a79efbc298530eb2fefe28a7116e986bd9083f68dfc3c13643021dd7bR5